### DISCUSSION TITLE
Question about Deposits and Deposit Refunds for QBO

### DISCUSSION THREAD

**Discussion by mjp90** [codatio/codat-docs#327] [Q&A] [answered]
Hi,

I'm currently building ERP integrations to QB Online and QB Desktop through Codat and have had some trouble understanding how to push my company's `Deposit` and `Deposit Return` objects. Currently, our Deposit object acts almost like a Credit Note, in that it's meant to represent an amount that can be applied to an Invoice for a particular customer, but it's separate from our Credit object and has slightly different behavior in our app.

For this reason, I've taken the approach where we push our Deposit objects as Credit Notes in QB, but then I'm a little lost on how we're able to push our concept of a "Deposit Return" to QB since it doesn't seem like you can return or reverse credit notes.

The only endpoint I've seen in the Codat Accounting API docs that has anything to do with Deposits is the Bank Account Transactions endpoint, but using that endpoint introduces additional complexity that we'd like to avoid.

Any guidance or friendly tips for possible workflows we can utilize here would be greatly appreciated!

**Comment by dcoplowe** [comment, marked as answer]
Hi Michael,

It sounds like it would be beneficial to have a call about this so that we can better understand your use case and ensure that you have all the information you need to build your solution. Are you happy for me to contact you via email to organise this?

Many thanks,

David

**Comment by mjp90** [comment]
Hi David,

I would greatly appreciate a call to go over some of my issues! My schedule
is very flexible so please let me know what works best for you.

Thanks,
Mike

**Reply by dcoplowe** [reply]
Hi Mike, 

One of my colleagues has reached out to organise a call. We look forward to working with you! 

Many thanks, 

David
